sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -130,7 +129,7 @@ def from_connection_string(cls, conn_str, **kwargs):
         :param str conn_str: The connection string of the Service Bus Namespace.
         :rtype: ~azure.servicebus.management.ServiceBusAdministrationClient
         """
-        endpoint, shared_access_key_name, shared_access_key, _, token, token_expiry = parse_conn_str(conn_str)
+        endpoint, shared_access_key_name, shared_access_key, _, token, token_expiry = _parse_conn_str(conn_str)
```

Comment:
it would be great if we could also update the one in `_mnagement_client._async.py` under `azure.servicebus.aio.management` namespace

Reply:
Hi @yunhaoling I will get on this, thank you for the review!

Reply:
@yunhaoling change has been made in latest commit, formally re-requesting review now :)